### PR TITLE
[misc] Update log messages format to match best practices

### PR DESCRIPTION
**Pull request by aubincleme**

### CODE REVIEW

File: xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-api/src/main/java/org/xwiki/notifications/internal/email/live/LiveMimeMessageIterator.java [hunk 1]
```diff
@@ -27,6 +27,7 @@
 import javax.inject.Inject;
 import javax.inject.Named;
 
+import org.apache.commons.lang.exception.ExceptionUtils;
```
Comment (sdumitriu) [inline xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-api/src/main/java/org/xwiki/notifications/internal/email/live/LiveMimeMessageIterator.java]: `lang3`

File: xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-api/src/main/java/org/xwiki/notifications/internal/rss/DefaultNotificationRSSManager.java [hunk 2]
```diff
@@ -26,6 +26,7 @@
 import javax.inject.Inject;
 import javax.inject.Singleton;
 
+import org.apache.commons.lang.exception.ExceptionUtils;
```
Comment (sdumitriu) [inline xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-api/src/main/java/org/xwiki/notifications/internal/rss/DefaultNotificationRSSManager.java]: `lang3` here as well.

Comment (aubincleme) [inline xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-api/src/main/java/org/xwiki/notifications/internal/rss/DefaultNotificationRSSManager.java]: Fixed in dac1ff6

File: xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-api/src/main/java/org/xwiki/notifications/internal/email/live/LiveMimeMessageIterator.java (hunk 1 above)
Comment (aubincleme) [inline xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-api/src/main/java/org/xwiki/notifications/internal/email/live/LiveMimeMessageIterator.java]: Fixed in dac1ff6

**Comment by aubincleme** [comment]
@sdumitriu We have to update the documentation [here](http://dev.xwiki.org/xwiki/bin/view/Community/JavaCodeStyle#HLoggingBestPractices) if we need to be _that_ specific.
Fixed in http://dev.xwiki.org/xwiki/bin/view/Community/JavaCodeStyle?viewer=changes&rev1=27.2&rev2=28.2

**Pull request merged**